backend/database-router.test.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-env mocha */
+
+import assert from 'node:assert'
+import RouterBackend from './database-router.js'
+import { cloneDeep, omit } from 'turtledash'
+
+const validConfig = {
+  'abc': {
```

Comment:
Maybe also a comment explaining that 'abc' is a prefix, for context to the reader?